canonical-web-and-design/snapcraft.io
# Remove p-fluid-grid

Fixes https://github.com/canonical-web-and-design/snapcraft.io/issues/2174

## Done
- Added custom col class with modifier
- Updated homepage p-fluid-grid use with the grid
- Updated first snap flow with the grid
- Updated /build with the grid
- Updated listing page screenshots with a custom grid

## QA
- Pull the branch
- `./run`
- Visit http://0.0.0.0:8004
- Resize the page up and down, make sure that the following flow to new lines properly:
  - `Official snaps from major publishers` logos
  - Language selection grid
  - Linux distro grid
- Visit http://0.0.0.0:8004/build
- Resize the page up and down
- Make sure distro icons flow to new lines properly
- Visit http://0.0.0.0:8004/first-snap/c
- Resize the page up and down
- Make sure the OS boxes flow to new lines properly
- Visit http://0.0.0.0:8004/snap_name/listing
- Resize the page up and down
- Make sure the media areas flow properly - add some screenshots, remove some, drag images on etc.
